### ISSUE TITLE
error: `outdent` expected but `<-` found

### ISSUE THREAD

**Issue by joan38** [scalacenter/scalafix#2111]
Since scalafix 0.13 we are seeing the following error:
```
/Users/jgoyeau/IdeaProjects/scalafix-0.4.2-issue/sessions-spark/src/main/scala/com/netflix/data/playback/sessions/util/sandbox/Sandbox.scala:13:13: error: `outdent` expected but `<-` found
          _ <- IO(println(show"${result}"))
            ^^
```

Here is a minimized repo of the issue:
https://github.com/joan38/scalafix-issue

Potentially related to https://github.com/scalacenter/scalafix/issues/1738 ?

Thanks

**Comment by bjaglin** [comment]
Thanks for the report & the repro!

It's a duplicate of https://github.com/scalacenter/scalafix/issues/2089, as I confirmed that the regression is fixed on master by running [the latest snapshot](https://scalacenter.github.io/scalafix/docs/users/installation.html#snapshot) on your source file:
```
cs launch ch.epfl.scala:scalafix-cli_2.13.15:0.13.0+35-c42a6159-SNAPSHOT -r sonatype:snapshots --main scalafix.cli.Cli -- -r RedundantSyntax --verbose --check --scala-version 3.3.4 Sandbox.scala
```

I will cut a release in a couple of weeks.

**Comment by joan38** [comment]
Ah! Thanks @bjaglin

**Comment by kubukoz** [comment]
I suppose this was https://github.com/scalameta/scalameta/issues/4008 ?

**Comment by bjaglin** [comment]
> I suppose this was [scalameta/scalameta#4008](https://github.com/scalameta/scalameta/issues/4008) ?

Nope, this particular issue was https://github.com/scalameta/scalameta/issues/3979: the corresponding fix https://github.com/scalameta/scalameta/pull/3980 was brought in via https://github.com/scalacenter/scalafix/pull/2103 since the repro was fixed with the corresponding snapshot 0.13.0+33-91361b87-SNAPSHOT

I assume scalafix master/snapshots are still subject to https://github.com/scalameta/scalameta/issues/4008 until a scalameta version is tagged and bumped here (unless it was a regression introduced in scalameta 4.11.0 that is [not merged yet](https://github.com/scalacenter/scalafix/pull/2110)? but i doubt so).

**Comment by bjaglin** [comment]
> Sounds like this is still an issue on 0.14.0:

@joan38 I can't reproduce - can you open a ticket with more details on the setup? it might be that scalafix is invoked with the wrong scala version and thus the wrong scalameta dialect is used (not supporting fewer braces)
